docs/PLUGINS.md

```diff
@@ -76,4 +76,10 @@ Created credentials may be reference via `${credentials.name1.username}` and `${
 
 ## org.starchartlabs.flare.merge-coverage-reports
 
-The merge-coverage-reports plug-in adds a task which generates a single XML Jacoco report which includes coverage information from reports in all sub-modules. The added task (`mergeCoverageReports`), depends on the `test` tasks of all sub-modules, and is added as a dependency of the `check` task of the root project
\ No newline at end of file
+The merge-coverage-reports plug-in adds a task which generates a single XML Jacoco report which includes coverage information from reports in all sub-modules. The added task (`mergeCoverageReports`), depends on the `test` tasks of all sub-modules, and is added as a dependency of the `check` task of the root project
+
+## org.starchartlabs.flare.source-jars
+
+The source-jars plug-in adds two tasks or type `Jar` - `sourcesJar`, which generates a jar file with the source files of the project, and `javadocJar`, which generates a jar file with the javadoc files of the project. These tasks are also added to the `archives` artifact configuration of the project.
+
+These jars are intended open-source projects to upload alongside binary artifacts to allow developers to trace through the associated code paths when debugging, and other development uses.
```

Comment:
This sentence is missing a word and kind of clunky anyway. I'd go with 
> These jars are intended to be uploaded alongside binary artifacts for open-source projects. They allow developers to trace through the associated code paths when debugging, and to view associated Javadoc documentation within their IDE.

Reply:
Yeah, I don't really know what happened to that, probably started with one thing and didn't fully revise it as I drafted. Went with your version, committed to the branch